integ-test/src/test/java/org/opensearch/sql/calcite/remote/CalciteExplainIT.java

```diff
@@ -1414,6 +1406,65 @@ public void testPushDownMinOrMaxAggOnDerivedField() throws IOException {
                 TEST_INDEX_ACCOUNT)));
   }
 
+  @Test
+  public void testExplainChartWithSingleGroupKey() throws IOException {
+    assertYamlEqualsIgnoreId(
+        loadExpectedPlan("explain_chart_single_group_key.yaml"),
+        explainQueryYaml(
+            String.format("source=%s | chart avg(balance) by gender", TEST_INDEX_BANK)));
+
+    assertYamlEqualsIgnoreId(
+        loadExpectedPlan("explain_chart_with_span.yaml"),
+        explainQueryYaml(
+            String.format("source=%s | chart max(balance) by age span=10", TEST_INDEX_BANK)));
+
+    assertYamlEqualsIgnoreId(
```

Comment:
query include row_split and col_split, should move to testExplainChartWithMultipleGroupKeys?